### ISSUE TITLE
Enable php extensions on docker

### ISSUE THREAD

**Issue by ivanrosolen** [herdphp/docker-phpqa#10]
In order to test all things, we need to add php extensions, like gd, to build and run tests

**Comment by jpjoao** [comment]
We also need ldap and mysqli (at least).

**Comment by brunoric** [comment]
I totally agree about the extensions. @ivanrosolen do you have any suggested list to enable? It should be simple change after having the list... just enable the desired ones here: https://github.com/herdphp/docker-phpqa/blob/master/src/Dockerfile#L93-L106.

**Comment by jpjoao** [comment]
The list is quite big: http://php.net/manual/en/extensions.alphabetical.php

**Comment by brunoric** [comment]
Hi @jpjoao the list is big indeed! But I can see that most of them are already enabled by default. About the non default ones, we can come up with a list for the ones that are more used or build the images with all of them.

**Comment by afk11** [comment]
Saw this repo and immediately thought it was for testing custom PHP extensions :P Has anyone been using the repo for this?
